From 8a8700fa7d348016192a5c1d2c70b053e5f7fc62 Mon Sep 17 00:00:00 2001
From: Ting-Ying Li <tingying.li@cypress.com>
Date: Wed, 3 Feb 2021 21:36:51 -0600
Subject: [PATCH 140/149] non-upstream: Revert "brcm80211: select
 WANT_DEV_COREDUMP conditionally for brcmfmac"

This reverts commit 885a93cf31106808cd3ded7c5e01e35fa495fdc5.

To fix DEV_COREDUMP function to be disabled in compat/Kconfig
of backports. The root cause is because the parsing rule of
lib/kconfig.py in backports will not consider the "select <sym>
if xxx" as a sel_line. In order to aviod affecting other driver's
configs, so we do not modify the sel_line logic in backports.

Signed-off-by: Ting-Ying Li <tingying.li@infineon.com>
Signed-off-by: Chi-hsien Lin <chi-hsien.lin@infineon.com>
---
 drivers/net/wireless/broadcom/brcm80211/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/Kconfig b/drivers/net/wireless/broadcom/brcm80211/Kconfig
index b239e067bba8..994353cd6a55 100644
--- a/drivers/net/wireless/broadcom/brcm80211/Kconfig
+++ b/drivers/net/wireless/broadcom/brcm80211/Kconfig
@@ -34,7 +34,7 @@ config BRCM_TRACING
 config BRCMDBG
 	bool "Broadcom driver debug functions"
 	depends on BRCMSMAC || BRCMFMAC
-	select WANT_DEV_COREDUMP if BRCMFMAC
+	select WANT_DEV_COREDUMP
 	---help---
 	  Selecting this enables additional code for debug purposes.
 
-- 
2.17.1

